lib/movie_list_widgets/movie_list.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+import '/screens/movie_home_page.dart';
+import '/constants/constants.dart';
+
+import '/data_classes/movie.dart';
+
+class MovieList extends StatelessWidget {
+  final List<Movie> moviesList;
+  static const double movieBoxSize = 125;
+  static const double titleSize = 20;
+  static const int overviewMaxLines = 3;
+  static const double elevationValue = 5;
+
+  const MovieList({
+    super.key,
+    required this.moviesList,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView.builder(
+      itemCount: moviesList.length,
+      itemBuilder: (
+        context,
+        index,
+      ) {
+        return Card(
+          elevation: elevationValue,
+          margin: const EdgeInsets.symmetric(
+            horizontal: Constants.paddingSize,
+            vertical: Constants.paddingSize,
+          ),
+          child: ElevatedButton(
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                SizedBox(
+                  height: movieBoxSize,
+                  width: movieBoxSize,
+                  child: Image(
+                    image: NetworkImage(
+                      moviesList[index].posterUrl,
+                    ),
+                  ),
+                ),
+                Expanded(
+                  child: Container(
+                    padding: const EdgeInsets.only(
+                      bottom: Constants.paddingSize,
+                    ),
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Padding(
+                          padding: const EdgeInsets.all(
+                            Constants.paddingSize,
+                          ),
+                          child: Text(
+                            moviesList[index].title,
+                            style: const TextStyle(
+                              fontSize: titleSize,
+                              fontWeight: FontWeight.bold,
+                            ),
+                          ),
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.only(
+                            left: Constants.paddingSize,
+                            right: Constants.paddingSize,
+                          ),
+                          child: Text(
+                            moviesList[index].overview,
+                            maxLines: overviewMaxLines,
+                            overflow: TextOverflow.ellipsis,
+                          ),
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.only(
+                            left: Constants.paddingSize,
+                            right: Constants.paddingSize,
```

Comment:
If you only need to modify left and right you can use `EdgeInsets.symmetric(horizontal: Constants.paddingSize)`

Reply:
fixed